jsonpickle/ext/numpy.py

```diff
@@ -1,6 +1,11 @@
 # -*- coding: utf-8 -*-
 
 from __future__ import absolute_import
+from builtins import *
```

Comment:
Have you run `make tox`?   I don't think there's a `builtins` module in python2.

```sh
$ python -V
Python 2.7.11

$ python -mbuiltins
/usr/bin/python: No module named builtins
```

Also, it's probably worth explicitly mentioning each symbol we want to import. We avoid blind `import *` for many reasons.  There should be a `make check` target you can use to run `flake8` on the code base, which should have flagged this line.